quarto/src/Entity/Game.php

```diff
@@ -456,14 +456,16 @@ public function getWinningPieces() : array
         $remainingPieces = $this->getRemainingPieces();
         $gridSize = count($this->grid);
         $j = 0;
-        foreach ($remainingPieces as $remainingPiece) {
-            if ($gridSize > 0) {
+        if ($gridSize > 0) {
```

Comment:
You can avoid indentations writing `if ($gridSize <= 0) { return; }`. It's more readable.